communication/examples/relay.rs

```diff
@@ -239,7 +241,7 @@ fn handle_input_line(
     relay_peer: PeerId,
     remote_target: &mut Option<PeerId>,
 ) {
-    let regex = Regex::new("SET\\s+\"(?P<target>[^\"]+)\"").unwrap();
+    let regex = Regex::new("SET\\s+\"(?P<target>[^\"]+)\"").expect("Invalid Reqex string.");
```

Comment:
BTW - how have we tested the regex?
Is this a potential ReDoS? (I worry about the `+` and not having an upper bound)...

Reply:
I changed it to use an upper bound of 64 chars.